theories/Makefile.coq.local

```diff
@@ -0,0 +1,15 @@
+# Local parameters for Coq's stdlib
+
+# %.vo: %.v $(PRELUDE_DEP)
+# 	$(SHOW)COQC $<
+# 	$(HIDE)$(TIMER) $(COQC) $(COQDEBUG) $(TIMING_ARG) $(COQFLAGS) $(COQLIBS) $< $(TIMING_EXTRA)
+
+PRELUDE_VO:=$(addprefix Init/, $(addsuffix .vo, Hexadecimal Number Notations Nat Byte Prelude Wf Peano Logic Logic_Type Tauto Decimal Numeral Datatypes Specif Ltac Tactics))
+
+COQMF_BOOT:=-boot
```

Comment:
I think it would be better to make `coq_makefile` understand `-boot` without `-arg`, so that it emits `COQMF_BOOT` itself (or else adds it to the relevant flag)